qdb/memqdb.go

```diff
@@ -427,7 +428,7 @@ func (q *MemQDB) GetTransferTx(ctx context.Context, key string) (*DataTransferTr
 
 	ans, ok := q.Transactions[key]
 	if !ok {
-		return nil, spqrerror.Newf(spqrerror.SPQR_TRANSFER_ERROR, "no tx with key %s", key)
+		return nil, nil
```

Comment:
the wat

Reply:
There's no need of an error if transaction does not exist